package.json

```diff
@@ -12,27 +12,24 @@
     "util-deprecate": "^1.0.1"
   },
   "devDependencies": {
+    "@babel/core": "^7.2.0",
+    "@babel/plugin-proposal-async-generator-functions": "^7.2.0",
+    "@babel/plugin-proposal-optional-catch-binding": "^7.2.0",
+    "@babel/plugin-transform-arrow-functions": "^7.2.0",
+    "@babel/plugin-transform-async-to-generator": "^7.2.0",
+    "@babel/plugin-transform-block-scoping": "^7.2.0",
+    "@babel/plugin-transform-computed-properties": "^7.2.0",
+    "@babel/plugin-transform-destructuring": "^7.2.0",
+    "@babel/plugin-transform-for-of": "^7.2.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.2.0",
+    "@babel/plugin-transform-shorthand-properties": "^7.2.0",
+    "@babel/plugin-transform-spread": "^7.2.0",
+    "@babel/plugin-transform-template-literals": "^7.2.0",
+    "@babel/polyfill": "^7.0.0",
+    "@babel/preset-env": "^7.2.0",
```

Comment:
Correction: `@babel/preset-env` is used, not by the main build script, but by the `update-browser-errors` npm script. Still, we're missing dependencies.

Reply:
Can you recommend some different way to do this? This is definitely my best, I feel using babel lead to fragile code anyway.

I've added the missing dependencies.

Reply:
Luckily, all the plugins that we use, are included in `@babel/preset-env`. So we can definitely simplify the setup. With some configuration to specify target environments (node and browser versions), which could reduce the number of code transformations. If we put that configuration in a `.babelrc`, it will be applied to both the build script and `update-browser-errors`.

We could even have two builds, one for node (with less code transformations, possibly better performance?) and one for browsers.

I can open an issue or PR (but might not get around to it until ~ a week from now).

Reply:
Would you be ok if we land this, ship it in a release, and then you send a PR to clean things up?